e2e/rook-test-framework/contracts/interface_rook_block_client.go

```diff
@@ -0,0 +1,11 @@
+package contracts
+
+type Irook_block interface {
```

Comment:
i would just call these interfaces Block, Client, FileSystem, etc. No need for prefixes